accesscontrol/jwt.go

```diff
@@ -204,6 +206,19 @@ func (j *JWT) Validate(req *http.Request) error {
 	}
 
 	ctx := req.Context()
+	if j.introspector != nil {
+		exp, _ := tokenClaims["exp"].(float64)
+		nbf, _ := tokenClaims["nbf"].(float64)
+		introspectionResponse, err := j.introspector.Introspect(ctx, tokenValue, int64(exp), int64(nbf))
+		if err != nil {
+			return err
+		}
+
+		if !introspectionResponse.Active() {
+			return errors.JwtTokenExpired.Message("token inactive")
```

Comment:
Is it ok to reuse `errors.JwtTokenExpired` here? The log message is "access control error: at: token inactive", but the message to the client is
```
Www-Authenticate: Bearer error="invalid_token", error_description="The access token expired"
                                                                                    ^^^^^^^
...
```

Reply:
If it should be token inactive we have to introduce a new error kind in `type_definitions.go`. (and hit generate)

Reply:
I added a new error kind.